File: tests/test_vulnerable.js
const crypto = require('crypto');
const NodeRSA = require('node-rsa');
const elliptic = require('elliptic');

// RSA - QUANTUM VULNERABLE
const { privateKey, publicKey } = crypto.generateKeyPairSync('rsa', {
    modulusLength: 2048,
});

// ECC - QUANTUM VULNERABLE
const ec = new elliptic.ec('secp256k1');
const key = ec.genKeyPair();

// DH - QUANTUM VULNERABLE
const dh = crypto.createDiffieHellman(2048);
dh.generateKeys();

// MD5 - QUANTUM VULNERABLE
const md5Hash = crypto.createHash('md5').update('password').digest('hex');

// SHA1 - QUANTUM VULNERABLE
const sha1Hash = crypto.createHash('sha1').update('data').digest('hex');

File: tests/test_vulnerable.py
from Crypto.PublicKey import RSA
from Crypto.Cipher import PKCS1_OAEP
from Crypto.Signature import pkcs1_15
import hashlib

# RSA key generation - QUANTUM VULNERABLE
key = RSA.generate(2048)
cipher = PKCS1_OAEP.new(key)

# MD5 hashing - QUANTUM VULNERABLE
data = b"secret password"
md5_hash = hashlib.md5(data).hexdigest()

# SHA1 hashing - QUANTUM VULNERABLE
sha1_hash = hashlib.sha1(data).hexdigest()

# ECDSA - QUANTUM VULNERABLE
from Crypto.PublicKey import ECC
ecc_key = ECC.generate(curve='P-256')